test/functional/feature_bip68_sequence.py

```diff
@@ -26,13 +26,14 @@
 # RPC error for non-BIP68 final transactions
 NOT_FINAL_ERROR = "non-BIP68-final (code 64)"
 
+DUMMY_P2WPKH_SCRIPT = CScript([b'a' * 21])
```

Comment:
Could add a comment here to explain what this is for less gifted people (like me). Maybe take this: https://github.com/bitcoin/bitcoin/pull/17108#issuecomment-541086969

Reply:
Okay, will add an explanation.